apps/organizations/models.py

```diff
@@ -320,18 +321,45 @@ def status(self) -> str:
     ##
     # Notifications
 
+    def _build_notification_message(self, subject, verb):
+        """Builds a message that will be displayed as an internal Slack notification.
+        """
+        msg = '{subject_name} ({subject_username}<{email}>) has {verb} an invitation for Organization <{organization_name}>'.format( # noqa: E501
+            verb=verb,
+            subject_name=subject.profile.get_full_name(),
+            subject_username=subject.username,
+            email=subject.email,
+            organization_name=self.organization.name,
+        )
+        return msg
+
     def build_notification_message__created(self):
         """Builds a message that will be displayed as an internal Slack notification
         when this invitation object is created.
         """
-        msg = '{invited_by_name} ({invited_by_username}) has sent an invitation for organization {organization_name} to {email}'.format(
-            invited_by_name=self.invited_by.profile.get_full_name(),
-            invited_by_username=self.invited_by.username,
-            organization_name=self.organization.name,
-            email=self.email,
-        )
+        msg = self._build_notification_message(self.invited_by, 'sent')
+        return msg
+    
+    def build_notification_message__accepted(self):
+        """Builds a message that will be displayed as an internal Slack notification
+        when this invitation object is accepted.
+        """
+        msg = self._build_notification_message(self.user, 'accepted')
         return msg
 
+    def build_notification_message__declined(self):
+        """Builds a message that will be displayed as an internal Slack notification
+        when this invitation object is declined.
+        """
+        msg = self._build_notification_message(self.user, 'declined')
+        return msg
+    
+    def build_notification_message__resent(self):
```

Comment:
- Put `__resent` function right under `__created`
- Let's get rid of all the docstrings on the `build_notification_message__*` functions -- it's too wordy, for not enough information. The function names are already sufficient in this case.

See: https://en.wikipedia.org/wiki/Self-documenting_code